storefront/modules/assortment/components/AssortmentBreadcrumbs.tsx

```diff
@@ -1,16 +1,18 @@
 import Link from 'next/link';
 import { useIntl } from 'react-intl';
+import ROUTES_CONFIG from '../../common/utils/getRouteConfig';
 
 const AssortmentBreadcrumbs = ({ paths = [], currentAssortment }) => {
   const intl = useIntl();
+
   return (
     <nav className="mt-2 mb-4 border-bottom pb-3">
       <Link href="/">
         <a className="mr-2 breadcrumb-link">
           {intl.formatMessage({ id: 'home' })}
         </a>
       </Link>
-      <Link href="/shop">
+      <Link href={`/${intl.formatMessage({ id: ROUTES_CONFIG.shop.slug })}`}>
```

Comment:
The developer should not need to import ROUTES_CONFIG for every Link he uses, this could be wrapped into a "LocalizedLink"